configs/templates/ufs-srw-dev/spack.yaml

```diff
@@ -31,7 +31,7 @@ spack:
   - parallelio@2.5.3
   - esmf@8.3.0b09
   - fms@2022.01
-  - bufr@11.7.0
+  - bufr@11.7.1
```

Comment:
As opposed to hpc-stack-dev (the outlier), ufs-srw-dev needs this update, thanks.

Reply:
left this in